File: src/main/java/com/hedvig/claims/events/ClaimStatusUpdatedEvent.java
package com.hedvig.claims.events;

import com.hedvig.claims.aggregates.ClaimsAggregate.ClaimStates;
import lombok.Value;

import java.time.LocalDateTime;

@Value
public class ClaimStatusUpdatedEvent {
  public String claimsId;
  public String userId;
  public LocalDateTime registrationDate;
  public ClaimStates state;
}

File: src/main/java/com/hedvig/claims/events/ClaimStatusUpdatedEvent.kt
package com.hedvig.claims.events

import com.hedvig.claims.aggregates.ClaimsAggregate.ClaimStates
import java.time.LocalDateTime

data class ClaimStatusUpdatedEvent(
  val claimsId: String,
  val userId: String,
  val registrationDate: LocalDateTime,
  val state: ClaimStates
)
